dataminer/Functions/DOM/DOM_DevOps/DOM_Editor/DOM_editor_attachments_networkshare.md

```diff
@@ -0,0 +1,48 @@
+---
+uid: DOM_editor_attachments_networkshare
+---
+
+# Configuring a network share to store attachments
+
+<!--RN43726-->
+
+From DataMiner 10.5.10/10.6.0 onwards, you can specify a UNC path and select credentials from the credentials library to store DOM attachments linked to DOM instances.
+
+By default, this feature is not enabled, and attachments are saved locally in the `C:\Skyline DataMiner\Documents` folder.
+
+1. Make sure a correct set of credentials of the type *Username and password credentials* is available in the credentials library (accessible in Cube via *System Center* > *System settings*).
+
+1. In the Automation app in DataMiner Cube, run the *DOM Editor* script.
+
+   You can find this script in the folder `DOM\DOM Main scripts`.
+
+   The script window will show a list of the available modules, as well as several buttons.
+
+1. Next to the module for which you want configure a network share, click *Edit*.
+
+   A window with four buttons will be displayed.
+
+   ![DOM Editor: edit module window](~/dataminer/images/DOM_Editor_edit_module.png)
+
+1. Click *Manager Settings*.
+
+1. Click *Network Share Attachment Settings* to open a new page to configure the settings.
+
+    ![DOM Editor: manager settings window](~/dataminer/images/DOM_Editor_managersettings.png)
+
+1. Select the correct set of credentials from the *Credential* dropdown.
```

Comment:
When "credentials" is used to refer to a set of identifiers for authentication, it should always be used in the plural form (as it refers to e.g. a username *and* a password). I've adjusted this wherever necessary in this text, except here, as this is a direct reference to the UI. Is this something that could be corrected in the UI as well? Then we could either correct this straight away here (since the difference with the current UI is minimal), or merge it as it is and correct it when the UI has been updated, as you prefer.